lesson24/src/js/main.js

```diff
@@ -73,4 +73,20 @@ submitButton.addEventListener("click", (e) => {
     window.location.href = "register-done.html";
 });
 
+const options = {
+    root: document.querySelector('[data-id="modal-inner"]'),
+};
+
+const removeDisabledForAgreeButton = (entries) => {
+    entries.forEach(entry => {
+        if(entry.isIntersecting){
```

Comment:
IMO
I'll share the points Sae-san shared with me :)

If there is only one target element, it is possible to use [Destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to retrieve only the first element

```suggestion
const removeDisabledForAgreeButton = ([entry]) => {
        if(entry.isIntersecting){
```

Reply:
Thank you for your comment!
So when you have one target, you can get it with a split assignment!
Good method and I learned a lot. 
I have modified.
https://github.com/nakagawamai/js-lesson/pull/26/commits/89567fd800ffe245b9399f1301f12b19577babf5